pilot/pkg/model/authentication.go

```diff
@@ -57,7 +57,7 @@ const (
 )
 
 // JwtKeyResolver resolves JWT public key and JwksURI.
```

Comment:
Add a comment about when this is initialized.

Reply:
Done. Thanks!